linchpin/provision/roles/ovirt/tasks/provision_ovirt_vms.yml

```diff
@@ -1,8 +1,34 @@
+---
+
+- name: "Provision ovirt_disk resources"
+  ovirt_disks:
+    state: "{{ state }}"
+    auth: "{{ ovirt_auth }}"
+    name: "{{ ovirt_resource_name }}-{{ disk_item['name'] }}"
+    size: "{{ disk_item['size'] }}"
+    format: "{{ disk_item['format'] }}"
+    storage_domain: "{{ disk_item['storage_domain'] }}"
+    interface: "{{ disk_item['interface'] | default(omit) }}"
+  with_items: "{{ res_def['disks'] | default([]) }}"
+  loop_control:
+    loop_var: disk_item
+  register: disk
+    
+- set_fact:
+    disks: []
+    
+- set_fact:
+    disks: "{{ disks + [{ 'name': disks_item['disk']['alias'] }] }}"
```

Comment:
should this be:
`     disks: "{{ disks | default([]) }} + {{ [{ 'name': disks_item['disk']['alias'] }] }}" `